webpack-subresource-integrity/src/util.ts

```diff
@@ -58,13 +66,20 @@ export const computeIntegrity = (
   return result;
 };
 
+const placeholderCache: Record<string, string> = {};
 export const makePlaceholder = (
   hashFuncNames: string[],
   id: string | number
 ): string => {
+  const cacheKey = hashFuncNames.join() + id;
+  if (placeholderCache[cacheKey]) {
+    return placeholderCache[cacheKey]!;
```

Comment:
The `!` assertion can be avoided here by doing something like:
```
const cachedPlaceholder = placeholderCache[cacheKey];
if (cachedPlaceholder) return cachedPlaceholder;
```

Reply:
Great idea! Will follow up